app/components/Contacts/Network/Network.js

```diff
@@ -263,17 +212,23 @@ class Network extends Component {
                 })}
 
               {currentChannels.length > 0 &&
-                currentChannels.map((channelObj, index) => {
-                  const channel = Object.prototype.hasOwnProperty.call(channelObj, 'channel')
-                    ? channelObj.channel
-                    : channelObj
-                  const pubkey = channel.remote_node_pub || channel.remote_pubkey
-                  const isSelected = selectedChannel === channel
-                  const status = channelStatus(channelObj)
+                currentChannels.map(channelObj => {
+                  const channel = channelObj.channel || channelObj
```

Comment:
Could you please add this comment here
// If this is a pending channel, the channel data will be stored under the `channel` key.